TesseractOCR/G8Tesseract.mm

```diff
@@ -287,18 +307,28 @@ - (void)setVariableValue:(NSString *)value forKey:(NSString *)key
      * _tesseract->SetVariable("language_model_penalty_non_freq_dict_word", "0");
      * _tesseract->SetVariable("language_model_penalty_non_dict_word ", "0");
      */
-
+    
     [self resetFlags];
 
+    if (!value) {
```

Comment:
Since you check `value`, shouldn't we also check `key`? The function could just return early if `key` isn't a valid string so that it's a no-op.